redis_info.py

```diff
@@ -15,6 +15,8 @@
 #
 # Authors:
 #   Garret Heaton <powdahound at gmail.com>
+# Contributors:
+#   Pierre Mavro <p.mavro@criteo.com> / Deimosfr <deimos at deimos.fr>
```

Comment:
Would be nice to link to https://github.com/powdahound/redis-collectd-plugin/graphs/contributors here too as there have been other contributors too